auth/auth-domain/src/test/java/shop/woowasap/auth/domain/PasswordTest.java

```diff
@@ -0,0 +1,65 @@
+package shop.woowasap.auth.domain;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import shop.woowasap.auth.domain.exception.PasswordValidateException;
+
+@DisplayName("유저 비밀번호 테스트")
+class PasswordTest {
+
+    @Nested
+    @DisplayName("비밀번호 정상 생성")
+    class WhenOfThenSuccess {
```

Comment:
이렇게 수정하는건 어떤가요?
`@Nested` 폴더링을 메소드 단위로 하면 다음과 같은 이점이 있습니다 
1. 메소드별로 빠진 테스트를 찾기 쉽습니다
2. 테스트가 구조화 되어있어서 출력결과 확인을 메소드 단위로 할 수 있음 -> 꼼꼼한 테스트와 메소드 명세로 이어집니다

이렇게 폴더링하고 of관련된 테스트는 하나의 Nested 하위에 Test로 몰아넣으면 됩니다!

+아마 다른팀원 분들도 Nested를 이 단위로 걸고있으실거에용~

```suggestion
    @Nested
    @DisplayName("Password.of 메소드는")
    class passwordOfMethod {
```

Reply:
감이 잘 안잡히면 내일 같이 해봐도 좋아요

Reply:
메소드 뒤에 '는'은 상위 구조만 볼때 부자연스러워서 뺐습니다. [521e3fb](https://github.com/woowasap/ASAP/pull/43/commits/521e3fb8b20ac5db8d6b103c47c759bfaf108b65)